src/design/plone/contenttypes/profiles/default/types/Pagina_Argomento.xml

```diff
@@ -50,6 +50,7 @@
     <element value="plone.leadimage" />
     <element value="collective.dexteritytextindexer"/>
     <element value="volto.blocks" />
+    <element value="design.plone.contenttypes.behavior.additional_help_infos"/>
```

Comment:
Occhio che poi compaia sempre dove ci si aspetta

Reply:
è un campo che sta in un tab da solo